route/task.js

```diff
@@ -143,24 +143,23 @@ module.exports = function(app) {
 			if (!task) {
 				return reply.response('Not Found').code(404);
 			}
-			if (process.env.NODE_ENV !== 'test') {
-				console.log('');
-				console.log(chalk.grey('Starting to run one-off task @ %s'), new Date());
-				console.log('Starting task %s', task.id);
-				const executed = await model.task.runById(request.params.id);
-				if (executed) {
-					console.log(chalk.green('Finished task %s'), task.id);
-				} else {
-					console.log(
-						chalk.red('Failed to finish task %s'),
-						task.id
-					);
-				}
+			console.log('');
```

Comment:
y u log nothing? 🤔

Reply:
oh wait that was in the original too 🤷‍♀

Reply:
yeah, that just made my scrat my head. I "think" it's to effectively do `console.writeNewLine()`